spec/batch_consumer_spec.rb

```diff
@@ -251,18 +251,11 @@ def consume_batch(_payloads, _metadata)
             'timestamp' => 2.minutes.ago.to_s, 'message_id' => 'two' }
         ]
 
-        allow(Deimos.config.logger).
-          to receive(:info)
-
-        expect(Deimos.config.logger).
-          to receive(:info).
-          with(hash_including(
-                 message_ids: [
-                   { key: 1, message_id: 'one' },
-                   { key: 2, message_id: 'two' }
-                 ]
-               )).
-          twice
+        allow(Deimos::Logging).to receive(:log_info)
+
+        expect(Deimos::Logging).
+          to receive(:log_info).
+          with(hash_including(payload_keys: ["1", "2"]))
```

Comment:
`it 'should log message identifiers'`. The expectation does not seem to test the `message_id` part and removes it.

I think the title should change if you say that this is the correct way in v2.